### PR TITLE
Refactor TwG color choices to a shared component.

### PR DESCRIPTION
**Pull request by makiost**
## Summary

Addresses issue:

- #5948 

## Relevant technical choices

I didn't add a story for the component, as it only renders a simple svg, and is covered by the TwG settings stories. If the reviewer deems it necessary, one can be added in 10 minutes 😄 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [x] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

**Comment by asvinb** [comment]
@makiost Code looks goods 👍 I do have a suggestion though. With the changes in the PR, we now have 2 properties, `colorPrimary` and `colorCode` which both have the same values. I suggest we get rid of `colorCode`, replacing it by `colorPrimary`
https://github.com/google/site-kit-wp/blob/ee7ba62a98721e9dcf70b97224750f4df453aa46/assets/js/modules/thank-with-google/util/settings.js#L52
Only [this code will need to be updated](https://github.com/google/site-kit-wp/blob/ee7ba62a98721e9dcf70b97224750f4df453aa46/assets/js/modules/thank-with-google/components/common/ColorRadio.js#L77) to use `colorPrimary`.

**Comment by makiost** [comment]
Hey @asvinb! Good point, fixed it now 👍

**Comment by asvinb** [comment]
LGTM

File: assets/js/modules/thank-with-google/components/common/ColorThemeChoice.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	);
+}
+
+ColorThemeChoice.propTypes = {
+	colorPrimary: PropTypes.string.isRequired,
+	colorSecondary: PropTypes.string.isRequired,
+};
+
+ColorThemeChoice.defaultProps = {
+	colorPrimary: '#E3F2FD',
```
Comment (techanvil) [inline assets/js/modules/thank-with-google/components/common/ColorThemeChoice.js:57]: Please can you add a comment to explain what these default colors are?

**Review by techanvil — APPROVED** [review]
Thanks @makiost! LGTM.

**Pull request merged**